contrib/git-lfs/template.py

```diff
@@ -0,0 +1,47 @@
+pkgname = "git-lfs"
+pkgver = "3.5.1"
+pkgrel = 0
+build_style = "go"
+make_dir = "bin"  # needed for tests
+make_build_args = [
+    "-ldflags=-X github.com/git-lfs/git-lfs/v3/config.Vendor=ChimeraLinux"
+]
+hostmakedepends = ["asciidoctor", "go"]
+checkdepends = ["bash", "curl", "git", "gmake", "perl"]
+depends = ["git"]
+pkgdesc = "Git extension for versioning large files"
+maintainer = "Subhaditya Nath <sn03.general@gmail.com>"
+license = "MIT"
+url = "https://git-lfs.com"
+source = f"https://github.com/git-lfs/git-lfs/releases/download/v{pkgver}/git-lfs-v{pkgver}.tar.gz"
+sha256 = "fc19c7316e80a6ef674aa4e1863561c1263cd4ce0588b9989e4be9461664d752"
+
+
+def post_build(self):
+    self.mkdir("man")
+    self.do(
+        "asciidoctor",
+        "-bmanpage",
+        "-Dman",
+        "-a",
+        f"mansource={pkgname} {pkgver}",
+        *[f for f in self.find("docs/man", "*.adoc")],
```

Comment:
use a normal loop

Reply:
IIRC a normal loop took MUCH longer, and that's why I did this optimization.

The normal loop took about 10-15 seconds on my system, whereas this one takes less than a second, IIRC.

Reply:
Just checked - 
- Normal loop takes ~8 seconds
- List unpacking takes ~0.5 seconds

:shrug:

Reply:
Should I keep this or switch to normal loop?

Reply:
ah that's true



i'd forgotten about them lol.